migrations/5_test_case_id_type.up.sql

```diff
@@ -0,0 +1,8 @@
+ALTER TABLE test_item
+    ALTER COLUMN test_case_id TYPE VARCHAR(256);
+
+ALTER TABLE test_item
+    ADD COLUMN test_case_hash INTEGER;
```

Comment:
can it be null?

Reply:
For now, not all agents can send us 'testCaseId' (it will be generated depending on 'uniqueId'), but when we will get rid of 'uniqueId' there will be no source for 'testCaseId' when it is not provided by an agent

Reply:
so, how would history be created ?

Reply:
Not null constraint with default 0 value added